lib/main.coffee

```diff
@@ -8,6 +8,9 @@ module.exports =
       scopes:
         '.source.jade':
           default: false
+    ignoreMarkdownLineBreak:
+      type: 'boolean'
```

Comment:
I think this needs a description or an improved name, it would take me awhile to think about whether checked or unchecked will trim trailing whitespace in markdown files since "ignoring the line break" means removing it in the context of this package.

Maybe something like:

* `keepTrailingTwoSpacesInMarkdownFiles` with a description like *Markdown treats two spaces at the end of a line as a line break*

Thoughts?